Samples/Client/SwiftClient/SwiftClient/VideoStreamViewController.swift

```diff
@@ -0,0 +1,640 @@
+            } else {
+                let peerId = parseIntHeader(r: response, name: "Pragma")
+                print("Message from: \(peerId): \(data)")
+                
+                if (peerId == myId) {
+                    handleServerNotification(data: data)
+                } else {
+                    handlePeerMessage(peerId: peerId, data: data)
+                }
+            }
+        }
+        
+        if hangingGet != nil {
+            hangingGet.cancel()
+            hangingGet = nil
+        }
+        
+        if myId != -1 {
+            startHangingGet()
+        }
+    }
+    
+    func startHangingGet() {
+        let urlString = URL(string: "\(server)/wait?peer_id=\(myId!)")!
+        var urlRequest = URLRequest(url: urlString)
+        urlRequest.setValue("Client", forHTTPHeaderField: "Peer-Type")
+        hangingGet = URLSession.shared.dataTask(with: urlRequest, completionHandler: hangingGetCallback)
+        hangingGet.resume()
+    }
+    
+    func startHeartBeat() {
+        if !heartBeatTimerIsRunning {
+            DispatchQueue.main.async {
+                self.heartBeatTimer = Timer.scheduledTimer(timeInterval: TimeInterval(self.heartbeatIntervalInSecs), target: self, selector: #selector(VideoStreamViewController.startHeartBeat), userInfo: nil, repeats: true)
+            }
+            heartBeatTimerIsRunning = true
+        }
+        let url = URL(string: "\(server)/heartbeat?peer_id=\(myId!)")!
+        var urlRequest = URLRequest(url: url)
+        urlRequest.setValue("Client", forHTTPHeaderField: "Peer-Type")
+        let heartbeatGet = URLSession.shared.dataTask(with: urlRequest)
+        heartbeatGet.resume()
+    }
+    
+    func signInCallback(data: Data?, response: URLResponse?, error: Error?) {
+        let response = response as! HTTPURLResponse
+        if response.statusCode == 200 && error == nil {
+            if let data = data, let stringData = String(data: data, encoding: String.Encoding.utf8) {
+                let peers = stringData.components(separatedBy: "\n")
+                myId = Int(peers[0].components(separatedBy: ",")[1])!
+                print("My id: \(myId)")
+                
+                for (index, peer) in peers.enumerated() {
+                    if index >= 1 && peer.characters.count > 0 {
+                        print("Peer \(index): \(peer)")
+                        let parsed = peer.components(separatedBy: ",")
+                        otherPeers[Int(parsed[1])!] = parsed[0]
+                    }
+                }
+                startHangingGet()
+                startHeartBeat()
+                updatePeerList()
+                
+                request = nil
+            }
+        }
+    }
+    
+    func signIn() {
+        let urlString = URL(string: "\(server)/sign_in?peer_name=\(localName)")!
+        var urlRequest = URLRequest(url: urlString)
+        urlRequest.setValue("Client", forHTTPHeaderField: "Peer-Type")
```

Comment:
this isn't used in our serverside code, i'm not sure how this got into a bunch of our clients 🙃 should probably remove it here since we have to start cleaning up somewhere 😸

Reply:
I will remove all Peer-Type related code.